src/vtk/DRCFilters/CMakeLists.txt

```diff
@@ -6,22 +6,22 @@ include_directories(${CMAKE_CURRENT_BINARY_DIR})
 find_package(OpenGL REQUIRED)
 
 find_package(Eigen REQUIRED)
-include_directories(${EIGEN_INCLUDE_DIRS})
+include_directories(SYSTEM ${EIGEN_INCLUDE_DIRS})
 
 set(sources
   vtkDepthImageUtils.cxx
   vtkGridSource.cxx
   vtkFrameWidget.cxx
   vtkFrameWidgetRepresentation.cxx
   vtkInteractorStyleTerrain2.cxx
-  vtkDepthImageProcessingPass.cxx
-  vtkEDLShading.cxx
-  vtkOBJImporter.cxx
+  #vtkDepthImageProcessingPass.cxx
+  #vtkEDLShading.cxx
+  #vtkOBJImporter.cxx
```

Comment:
Cruft? (Actually, I believe some of this should be enabled or not depending on the VTK back-end... or did we decide that only OpenGL2 is supported?)